### ISSUE TITLE
Support for django.models.JSONField

### ISSUE THREAD

**Issue by HT-Moh** [justquick/django-activity-stream#472]
Hi 
As I am using the last Django version which has `JSONField` integrated, and it will be great if we add the support for it to prevent adding external library. 
I will suggest the following change on `jsonfield.py` file : 

```
if USE_JSONFIELD:
    try:
        from models import JSONField

        DataField = JSONField
    except ImportError as err:
        try:
            from jsonfield_compat import JSONField, register_app

            DataField = JSONField
        except ImportError as err:
            try:
                from django_mysql.models import JSONField

                DataField = JSONField

            except ImportError:
                raise ImproperlyConfigured(
                    "You must either install django-jsonfield + "
                    "django-jsonfield-compat, or django-mysql as an "
                    "alternative, if you wish to use a JSONField on your "
                    "actions"
                )

```

**Comment by auvipy** [comment]
please provide a full PR

**Comment by lociii** [comment]
This has been resolved in the now merged #474
